File: 2023 czerwiec/INF.04-03-23.06-SG & INF.04-01-23.06-SD/Aplikacja Konsolowa/Python/main.py
"""
    nazwa klasy: Film
    pola: title - przechowuje tytuł filmu
    loanCount - ilość wypożyczonych filmów
    metody: titleSetter, brak zwracanej wartości – ustawia tytuł filmu na podany
            titleGetter, zwraca nazwę filmu - zwraca nazwę filmu
            loanCountGetter, zwraca ilość wypożyczonych filmów - zwraca ilość wypożyczonych filmów
            loanCountIncrementer, brak zwracanej wartości - inkrementuje ilośc wypożyczonych filmów 
    informacje: klasa opisująca film
    autor: 00000000000
"""
class Film:
    def __init__(self):
        self._title = ""
        self._loanCount = 0
    
    def titleSetter(self, newTitle):
        self._title = newTitle
        
    def titleGetter(self):
        return self._title
    
    def loanCountGetter(self):
        return self._loanCount
    
    def loanCountIncrementer(self):
        self._loanCount += 1
        
film1  = Film()

print("Tytuł: ", film1.titleGetter())
print("Liczba wypożyczeń: ", film1.loanCountGetter())

film1.titleSetter("Harry Poter")
film1.loanCountIncrementer()

print("Tytuł: ", film1.titleGetter())
print("Liczba wypożyczeń: ", film1.loanCountGetter())

File: 2023 czerwiec/INF.04-03-23.06-SG & INF.04-01-23.06-SD/Aplikacja Konsolowa/C++/main.cpp
#include <iostream>

using namespace std;
/*
    * nazwa klasy: Film
    * pola: title - przechowuje tytuł filmu
    * loanCount - ilość wypożyczonych filmów
    * metody: titleSetter, brak zwracanej wartości – ustawia tytuł filmu na podany
    *         titleGetter, zwraca nazwę filmu - zwraca nazwę filmu
    *         loanCountGetter, zwraca ilość wypożyczonych filmów - zwraca ilość wypożyczonych filmów
    *         loanCountIncrementer, brak zwracanej wartości - inkrementuje ilośc wypożyczonych filmów 
    * informacje: klasa opisująca film
    * autor: 00000000000
*/
class Film
{
protected:
    string title = "";
    int loanCount = 0;

public:
    void titleSetter(string newTitle)
    {
        title = newTitle;
    }

    string titleGetter()
    {
        return title;
    }

    int loanCountGetter()
    {
        return loanCount;
    }

    void loanCountIncrementer()
    {
        loanCount++;
    }
};

int main()
{
    Film film1;

    cout << "Tytuł: " << film1.titleGetter() << endl;
    cout << "Liczba wypożyczeń: " << film1.loanCountGetter() << endl;

    film1.titleSetter("Harry Poter");
    film1.loanCountIncrementer();

    cout << "Tytuł: " << film1.titleGetter() << endl;
    cout << "Liczba wypożyczeń: " << film1.loanCountGetter() << endl;
}